mne_bids/write.py

```diff
@@ -1009,6 +1009,10 @@ def write_raw_bids(raw, bids_path, events_data=None,
             recording date will be overwritten as well. If True, keep subject
             information apart from the recording date.
 
+    convert : bool
+        Whether to convert files to RECOMMENDED BIDS format for that
+        datatype. For example, if iEEG, then the RECOMMENDED format is
+        BrainVision, so files would be converted to have extension ``.vhdr``.
```

Comment:
```suggestion
        Whether to convert files to the recommended BIDS format for that
        datatype. For example, for iEEG the recommended format is
        BrainVision, so the data would be converted to ``.vhdr`` files.
```

Reply:
Sorry this is meant to address a different PR... Idk how this wound up here :/